### PR TITLE
Fix compile signature for debugenginehost in Desktop.* builds

### PR DESCRIPTION
**Pull request by pieandcakes**
This is to fix inserting custom built MIEngine into VSCode cpptools
because it was using a dev signature which didn't match what the actual
dll that it was used in this scenario

### CODE REVIEW

**Comment by pieandcakes** [comment]
this is to fix https://github.com/Microsoft/vscode-cpptools/issues/1642

**Comment by gregg-miskelly** [comment]
>   </PropertyGroup>

Delete this part of the code

---
Refers to: src/DebugEngineHost.Stub/DebugEngineHost.Stub.csproj:74 in e6bf660. [](commit_id = e6bf660df1d8d8155c130de3de247db81265430e, deletion_comment = False)

**Comment by pieandcakes** [comment]
@gregg-miskelly I can't. VS automatically keeps adding it back when I open the sln.

**Comment by pieandcakes** [comment]
@dotnet-bot test this please

**Comment by pieandcakes** [comment]
@chuckries Can you verify the .groovy file?

**Comment by pieandcakes** [comment]
@dotnet-bot test ci please

File: netci.groovy [hunk 1]
```diff
@@ -15,8 +15,9 @@ def branch = GithubBranchName
         
         Utilities.setMachineAffinity(newJob, 'Windows_NT', 'latest-or-auto')
         Utilities.standardJobSetup(newJob, project, isPR, "*/${branch}")
-        Utilities.addXUnitDotNETResults(newJob, 'bin/**/TestResults.xml')
-        
+        if (isPr != 'Desktop.Debug' && isPr != 'Desktop.Release') {
```
Comment (chuckries) [inline netci.groovy]: isPr will be true/false, not a string. you want `configuration` here.

**Comment by pieandcakes** [comment]
@dotnet-bot test ci please

**Comment by pieandcakes** [comment]
@dotnet-bot test ci please

**Pull request merged**